File: phasegateone/add.py
digit = int(input("input a three digit number: "))
if digit>0 and digit<1000:
	number = digit
	digit1 = number%10
	digit2 = number//10
	digit3 = digit2%10
	digit4 = digit2//10
	add = digit1+digit3+digit4
	print(add)
else:
	print("ERROR")

File: phasegateone/Add.java
import java.util.Scanner; 
public class Add{
	public static void main(String[] args){
		Scanner input = new Scanner(System.in);
		System.out.println("input a number: "); 
		int digit = input.nextInt();

		if (digit>0 || digit<1000){
			int number = digit;
			int digit1 = number%10;
			int digit2 = number/10;
			int digit3 = digit2%10;
			int digit4 = digit2/10;
	
			int add = digit1+digit3+digit4;
			System.out.print(add);
		}else{
			System.out.print("ERROR");
		}
	}
}